.gitattributes

```diff
@@ -19,6 +19,7 @@
 *.pptx binary
 *.fig binary
 *.mat binary
+*.mexw64 binary
```

Comment:
DCIMatlabR2017b6_1.mexw64 became corrupted with one of the last commits - prob. because it was not handled as binary.